deploy/stacks/pipeline.py

```diff
@@ -604,6 +604,7 @@ def set_backend_stage(self, target_env, repository_name):
                 commit_id=self.image_tag,
                 vpc_id=target_env.get('vpc_id'),
                 vpc_endpoints_sg=target_env.get('vpc_endpoints_sg'),
+                vpc_restricted_nacls=target_env.get('vpc_restricted_nacl', False),
```

Comment:
We should add this param to `template.cdk.json` and create a PR / GitHub issue to update  [git pages](https://awslabs.github.io/aws-dataall/deploy-aws/ ) (for `tooling_vpc_restricted_nacl`) :)

Reply:
Added the above and new PR to `gh-pages` #631